lib/ansible/executor/task_executor.py

```diff
@@ -134,7 +134,10 @@ def _clean_res(res):
                 if isinstance(res, UnsafeProxy):
                     return res._obj
                 elif isinstance(res, binary_type):
-                    return to_text(res, errors='surrogate_or_strict')
+                    try:
+                        return to_text(res, errors='surrogate_or_strict')
+                    except:
+                        return bytes(res)
```

Comment:
I think this will just lead to more errors later (as we're now returning either text or bytes rather than always returning text.

@jimi-c is it okay to replace bytes with unicode replacement characters here?  If wo, we can just use to_text's default error handler.